### PR TITLE
Maze Game

### PR DESCRIPTION
**Pull request by khushigarg1**
## 🛠️ Issue (Number)
#536 

Closes #

# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

I made this game using HTML, CSS, JAVASCRIPT in which i used css for background color tranformation at diff time interval then i used some case fristaly we have to select role of the game (easy,medium,hard,pro) then start the game then you will a board that will be show like one green ball and one flag i.e. ending point.
you have to go through the path and have to react at that point by left, right, bottm adn top arrows keys. IF you will react your goal then there will be shown one screen i.e. leaderbaord taht will show congratulations and your moves etc.

I used two js libraries :--
jQuery-3.3.1.min.js:- It is a fast, small, and feature-rich JavaScript library that makes things like HTML document traversal and manipulation, event handling, and animation much simpler with an easy-to-use API that works across a multitude of browsers.

jquery-touchSwipe.min.js:- It enables a developer to determine whether a user has swiped in a particular direction and provides the ability to handle swipe events such as swipe left, swipe right, swipe up, and swipe down.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
Screenshot (93)
Screenshot (94)
Screenshot (95)
Screenshot (96)
Screenshot (97)

### CODE REVIEW

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
add a readme.md file for your project

**Review by zugzwang03 — CHANGES_REQUESTED** [review]
images are not visible in the readme file

**Review by Arun9739 — CHANGES_REQUESTED** [review]
Create a separate image folder for images and put all image over there.

File: Projects/maze Game/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
```
Comment (wokla23006) [inline Projects/maze Game/index.html:3]: <script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-2270952767365068"
     crossorigin="anonymous"></script>

File: Projects/maze Game/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta charset="utf-8" />
+    <!-- <link rel="stylesheet" href="css/index.css" /> -->
```
Comment (wokla23006) [inline Projects/maze Game/index.html:5]: <script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-2270952767365068" crossorigin="anonymous"></script>

**Pull request closed without merging**